### ISSUE TITLE
 makeTrackValidationPlots.py crashes

### ISSUE THREAD

**Issue by VinInn** [cms-sw/cmssw#34420]
was ok in CMSSW_12_0_X_2021-06-30-2300 (that is still around)
```
Traceback (most recent call last):
  File "/cvmfs/cms-ib.cern.ch/week0/slc7_amd64_gcc900/cms/cmssw-patch/CMSSW_12_0_X_2021-07-08-2300/bin/slc7_amd64_gcc900/makeTrackValidationPlots.py", line 146, in <module>
    main(opts)
  File "/cvmfs/cms-ib.cern.ch/week0/slc7_amd64_gcc900/cms/cmssw-patch/CMSSW_12_0_X_2021-07-08-2300/bin/slc7_amd64_gcc900/makeTrackValidationPlots.py", line 76, in main
    val.doPlots(trk, plotterDrawArgs=drawArgs, **kwargs_tracking)
  File "/cvmfs/cms-ib.cern.ch/week0/slc7_amd64_gcc900/cms/cmssw-patch/CMSSW_12_0_X_2021-07-08-2300/python/Validation/RecoTrack/plotting/validation.py", line 1260, in doPlots
    self._doPlotsForPlotter(plotter, sample, **kwargs)
  File "/cvmfs/cms-ib.cern.ch/week0/slc7_amd64_gcc900/cms/cmssw-patch/CMSSW_12_0_X_2021-07-08-2300/python/Validation/RecoTrack/plotting/validation.py", line 1268, in _doPlotsForPlotter
    plotterInstance = plotter.readDirs(*self._openFiles)
  File "/cvmfs/cms-ib.cern.ch/week0/slc7_amd64_gcc900/cms/cmssw-patch/CMSSW_12_0_X_2021-07-08-2300/python/Validation/RecoTrack/plotting/plotting.py", line 3011, in readDirs
    return PlotterInstance([plotterItem.readDirs(files) for plotterItem in self._plots])
  File "/cvmfs/cms-ib.cern.ch/week0/slc7_amd64_gcc900/cms/cmssw-patch/CMSSW_12_0_X_2021-07-08-2300/python/Validation/RecoTrack/plotting/plotting.py", line 3011, in <listcomp>
    return PlotterInstance([plotterItem.readDirs(files) for plotterItem in self._plots])
  File "/cvmfs/cms-ib.cern.ch/week0/slc7_amd64_gcc900/cms/cmssw-patch/CMSSW_12_0_X_2021-07-08-2300/python/Validation/RecoTrack/plotting/plotting.py", line 2930, in readDirs
    return PlotterFolder(self._name, self._possibleDirs, subFolders, self._plotFolder, self._fallbackNames, self._fallbackDqmSubFolders, self._tableCreators)
  File "/cvmfs/cms-ib.cern.ch/week0/slc7_amd64_gcc900/cms/cmssw-patch/CMSSW_12_0_X_2021-07-08-2300/python/Validation/RecoTrack/plotting/plotting.py", line 2741, in __init__
    self._dqmSubFolders.sort(key=lambda sf: sf.subfolder)
AttributeError: 'dict_values' object has no attribute 'sort'
```

**Comment by VinInn** [comment]
@makortel @mtosi @vmariani

**Comment by mmusich** [comment]
due to https://github.com/cms-sw/cmssw/commit/a55870864bd5fa5634eab0a5d24050fae945ab5b ?
@davidlange6

**Comment by makortel** [comment]
I'm taking a look. That particular issue is easy to fix, but there can be further issues.

**Comment by makortel** [comment]
assign dqm

**Comment by makortel** [comment]
Fixed in #34430.

**Comment by jfernan2** [comment]
+1
For the records
